### DISCUSSION TITLE
Missing emojis

### DISCUSSION THREAD

**Discussion by iml0kesh** [florisboard/florisboard#1551] [Q&A]
it's look like some of the emojis have been missing on the Florisboard..

**Comment by tsiflimagas** [comment]
As I can see, Patrick has already started working on adding emojis back to FlorisBoard (the emoji screen was removed on some of the 0.3.14 betas). So, on the next iteration, you'll see a more complete emoji section. If not, then you may report which emojis are missing exactly.

**Reply by patrickgold** [reply]
I am currently working locally on re-adding emojis, and the emoji view should be back in a few days. What [iml0kesh](https://github.com/iml0kesh) is referring to I think is that FlorisBoard in 0.3.13 removed icons from the emoji view which were not supported by the system font. This was done because else it would just show a tofu box, which would make no sense at all.
